packages/fxa-payments-server/public/index.html

```diff
@@ -18,7 +18,7 @@
 
 <body>
     <div id=fox-logo></div>
-    <div id=stage>
+    <div id=stage style="opacity: 1">
```

Comment:
I'm surprised CSP rules let you do this.

Reply:
Hmm, I think this is a temporary hack - the app should apply this itself. Additionally, I think this payment server might not be applying good CSP rules yet